app/internal/timer.py

```diff
@@ -0,0 +1,21 @@
+from app.routers.event import sort_by_date
+from app.routers.user import get_all_user_events
+from sqlalchemy.orm import Session
+from app.database.models import Event
+
+
+def get_next_user_event(session: Session, user_id: int) -> Event:
```

Comment:
It's `Optional[Event]` (you might return None)

Reply:
Done